core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -800,7 +802,7 @@ private Name() {} // prevent instantiation
    * @return whether the input is a valid property name
    */
   public static boolean isValid(String keyStr) {
-    return KEYS_MAP.containsKey(keyStr);
+    return KEYS_MAP.containsKey(keyStr) || keyStr.startsWith("fs");
```

Comment:
@gilv you don't need to add the prefix check here and `KEYS_MAP.containsKey(keyStr)` alone should be sufficient I believe. The reason is after you add `STOCATOR_SCHEME_LIST` into the `enum` `PropertyKey`,  this property key `STOCATOR_SCHEME_LIST` will be added to `KEYS_MAP` automatically when `KEYS_MAP` initialized